src/api/app/components/build_results_monitor_component.html.haml

```diff
@@ -23,20 +23,28 @@
                 = render partial: 'webui/shared/build_status_count_badge', locals: { category: key, count: value.to_s }
 
       .accordion-collapse.collapse{ id: "collapse-#{package_name.gsub(':', '_')}", class: show }
-        .list-group.list-group-flush.rounded-bottom
+        .rounded-bottom.build-result-table{ class: "build-result-table-#{filtered_architecture_names.length}" }
```

Comment:
```suggestion
        .rounded-bottom.build-result-table
```
It is not necessary to use this class if we use `grid-template-columns: 1fr repeat(20, auto);` and remove the `@for ...` loop.